### PR TITLE
Let's support ≤ 6 version.

### LINKED ISSUE #6: Elasticsearch ≤ 6 is not supported
With the current configuration, Elasticserarch will never boot with versions lower than 7, because of the different configuration of the discovery algorithm.

Further reading: https://www.elastic.co/blog/a-new-era-for-cluster-coordination-in-elasticsearch

### PR DESCRIPTION
**Pull request by miyataka**
- fix https://github.com/elastic/elastic-github-actions/issues/6
- please check this [CI result](https://github.com/miyataka/elastic-github-actions/actions/runs/106590407)

### CODE REVIEW

**Comment by miyataka** [comment]
I have signed the CLA now.
How to rerun the CLA agreement checker bot?

**Comment by miyataka** [comment]
@delvedor 
Would you please review this PR?

File: .github/workflows/elasticsearch.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ jobs:
   run-action:
     name: Start Elasticsearch
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        elasticsearch: ["5.6.16", "6.8.9", "7.7.0", "8.0.0-SNAPSHOT"]
```
Comment (delvedor) [inline .github/workflows/elasticsearch.yml]: Can you please use the following versions?
```
6.8-SNAPSHOT
7.x-SNAPSHOT
8.0-SNAPSHOT
```

Elasticsearch 5 is [no longer supported](https://www.elastic.co/support/eol) :)

**Review by delvedor — COMMENTED** [review]
Great work! Just a nit.

File: .github/workflows/elasticsearch.yml (hunk 1 above)
Comment (miyataka) [reply .github/workflows/elasticsearch.yml]: Thank you your comment.
I'll fix it.
please wait a few minutes.

**Comment by miyataka** [comment]
fixed.
and this is [CI result](https://github.com/miyataka/elastic-github-actions/runs/689630145?check_suite_focus=true)

File: elasticsearch/run-elasticsearch.sh [hunk 2]
```diff
@@ -7,30 +7,67 @@ if [[ -z $STACK_VERSION ]]; then
...
+      --env "bootstrap.memory_lock=true" \
+      --env "ES_JAVA_OPTS=-Xms1g -Xmx1g" \
+      --env "xpack.security.enabled=false" \
+      --ulimit nofile=65536:65536 \
+      --ulimit memlock=-1:-1 \
+      --publish "${port}:9200" \
+      --detach \
+      --network=elastic \
+      --name="es${node}" \
+      docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION}
```
Comment (delvedor) [inline elasticsearch/run-elasticsearch.sh]: You can remove this check, as we don't support Elasticsearch 5 :)

Comment (miyataka) [reply elasticsearch/run-elasticsearch.sh]: That's right.
fixed.
https://github.com/miyataka/elastic-github-actions/actions/runs/109452371

**Comment by delvedor** [comment]
Almost there! :)

The Elasticsearch 6.x configuration must be updated to accept multiple nodes, in the multi-node cluster test with Elasticsearch 6.x, there is only one node: https://github.com/miyataka/elastic-github-actions/runs/689663048?check_suite_focus=true#step:8:9

**Comment by miyataka** [comment]
I see...
please wait..

**Comment by miyataka** [comment]
@delvedor 
Sorry, late reply. I've fixed review point.

**Comment by delvedor** [comment]
Looks ok!
Can you merge the upstream master branch in your fork?
So we can also see the build result here in the pr :)

**Comment by miyataka** [comment]
> Looks ok!
> Can you merge the upstream master branch in your fork?
> So we can also see the build result here in the pr :)

Yeesssssssss!
I've merged.

**Comment by miyataka** [comment]
ping @delvedor

**Review by delvedor — APPROVED** [review]
Great work!

**Pull request merged**